packages/mail/script/generateTextTemplates.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+const { promises: fs } = require('fs')
+const htmlToText = require('html-to-text')
+const path = require('path')
+const Promise = require('bluebird')
+const recursive = require('recursive-readdir')
+
+const run = async () => {
+  const files = await recursive(
+    path.resolve(`${__dirname}/../templates`),
+    [file => path.extname(file) !== '.html']
+  )
```

Comment:
add an `argv` option to only select one template

Reply:
Done.